src/observer/consume.py

```diff
@@ -61,9 +61,9 @@ def consume(endpoint, usrparams={}):
 def default_idfn(row):
     return row['id']
 
-def upsert(id, content_type, content):
+def upsert(iid, content_type, content):
```

Comment:
`iid` stands for?

Reply:
(https://en.wikipedia.org/wiki/Independent_and_identically_distributed_random_variables)

Reply:
it's whatever your imagination wants it to be ... which is apparently a sequence of independent and identically distributed random values and not shorthand for `item-id`, like it was in my head.

![iid](https://user-images.githubusercontent.com/2142748/37314026-c4134432-26a1-11e8-9fc7-78e9b1eb6d20.jpeg)